applications/asset_tracker_v2/src/ext_sensors/Kconfig

```diff
@@ -9,6 +9,13 @@ menuconfig EXTERNAL_SENSORS
 
 if EXTERNAL_SENSORS
 
+config EXTERNAL_SENSORS_ACTIVITY_DETECTION_AUTO
+	bool "Set accelerometer trigger handler on initialization"
+	help
+	  Enable this option to start activity detection when the library is initialized.
+	  If the option is disabled, the accelerometer trigger handler must manually be enabled to
```

Comment:
What do users have to do, if the disable it?

Reply:
The have to call the function ext_sensors_accelerometer_trigger_callback_set(true).